instaclustr/resource_cluster.go

```diff
@@ -1235,6 +1240,20 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func deleteAttributesConflictWithDataCentres(d *schema.ResourceData) error {
+	clusterResource := resourceCluster()
```

Comment:
How nasty would it be to mock out `resourceCluster()` here so that we can stick some unit tests around this?

Reply:
A unit test has been added for this function